data/transactions/logic/doc.go

```diff
@@ -161,11 +179,12 @@ var opDocExtraList = []stringString{
 	{"bz", "See `bnz` for details on how branches work. `bz` inverts the behavior of `bnz`."},
 	{"b", "See `bnz` for details on how branches work. `b` always jumps to the offset."},
 	{"intcblock", "`intcblock` loads following program bytes into an array of integer constants in the evaluator. These integer constants can be referred to by `intc` and `intc_*` which will push the value onto the stack. Subsequent calls to `intcblock` reset and replace the integer constants available to the script."},
-	{"bytecblock", "`bytecblock` loads the following program bytes into an array of byte string constants in the evaluator. These constants can be referred to by `bytec` and `bytec_*` which will push the value onto the stack. Subsequent calls to `bytecblock` reset and replace the bytes constants available to the script."},
+	{"bytecblock", "`bytecblock` loads the following program bytes into an array of byte-array constants in the evaluator. These constants can be referred to by `bytec` and `bytec_*` which will push the value onto the stack. Subsequent calls to `bytecblock` reset and replace the bytes constants available to the script."},
 	{"*", "Overflow is an error condition which halts execution and fails the transaction. Full precision is available from `mulw`."},
```

Comment:
would love to see some extended example doc of setbit in the opDocExtra

Reply:
How about:
```
	{"getbit", "bit indexing begins with low-order bits in integers. Bit 4 is the 1 in int 0x0010. Indexing begins in the first bytes of a byte-string (as seen in getbyte and substring). Bits 0 through 11 are 1 in byte 0xfff000000000"},
	{"setbit", "see explanation of bit ordering in getbit"},
```

Reply:
I like putting the docs in setbit because starting with 0x0000 it's really clear to see what happens.

Reply:
How about this:
```
{"getbit", "see explanation of bit ordering in setbit"},
{"setbit", "bit indexing begins with low-order bits in integers. Setting bit 4 to 1 on the integer 0 yields 16 (`int 0x0010`, or 2^4). Indexing begins in the first bytes of a byte-string (as seen in getbyte and substring). Setting bits 0 through 11 to 1 in a 4 byte-array of 0s yields `byte 0xfff00000`"},
```

Reply:
Yup. This is unambiguous; thanks.